extension/android/benchmark/app/src/main/java/org/pytorch/minibench/LlmBenchmarkActivity.java

```diff
@@ -97,6 +124,7 @@ class StatsInfo {
   long generateStart;
   long generateEnd;
   String tokens;
+  String name;
 
```

Comment:
Maybe we can add a "loadStatus" for https://github.com/pytorch/executorch/blob/5c5499ad22750b5408c3107e8df12b46baf2819d/extension/android/benchmark/app/src/main/java/org/pytorch/minibench/LlmBenchmarkActivity.java#L63 in case of failure